app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -568,4 +578,38 @@ class SmokeTest {
             verifyUrl("webcompat.com/issues/new")
         }
     }
+
+    @Test
+    /* Verifies downloads in the Downloads Menu:
+      - downloads appear in the list
+      - deleting a download from device storage, removes it from the Downloads Menu too
+    */
+    fun manageDownloadsInDownloadsMenuTest() {
+        val downloadWebPage = TestAssetHelper.getDownloadAsset(mockWebServer)
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(downloadWebPage.url) {
+            mDevice.waitForIdle()
+        }
+
+        downloadRobot {
+            verifyDownloadPrompt()
+        }.clickDownload {
+            mDevice.waitForIdle()
+            verifyDownloadNotificationPopup()
+        }
+
+        browserScreen {
+        }.openThreeDotMenu {
+        }.openDownloadsManager {
+            waitForDownloadsListToExist()
+            verifyDownloadedFileName("Globe.svg")

```

Comment:
We can probably just create a local variable with the filename so we only need to change it once if we change the filename in the future

Reply:
sure, good point.